assets/js/googlesitekit/datastore/ui/ui.js

```diff
@@ -22,12 +22,41 @@
 import invariant from 'invariant';
 import isPlainObject from 'lodash/isPlainObject';
 
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_UI } from './constants';
+
 const SET_VALUES = 'SET_VALUES';
 const SET_VALUE = 'SET_VALUE';
 
-export const initialState = {};
+export const initialState = {
+	useInViewResetCount: 0,
+};
 
 export const actions = {
+	/**
+	 * Resets all `useInView` hooks that have the `sticky` param set to `true`.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @return {Object} Redux-style action.
+	 */
+	*resetInView() {
```

Comment:
Let's rename it to `resetInViewCount` for consistency with the selector define below: 
```suggestion
	*resetInViewCount() {
```

Reply:
I like the action name being `resetInView` because it more clearly shows what the action does. I'll change the selector instead, but maybe even it makes more sense as `resetInViewHook` and `getInViewHook`. I'll try that and see what you think 🙂